lib/Policy.js

```diff
@@ -0,0 +1,44 @@
+/**
+ * Trails Policy Class.
+ */
+module.exports = class TrailsPolicy {
+
+  constructor (app) {
+    Object.defineProperty(this, 'app', {
+      enumerable: false,
+      writable: false,
+      value: app
+    })
+  }
+
+  /**
+   * Policy configuration
+   */
+  static config () {
```

Comment:
what's the purpose of this static method ?

Reply:
This is mostly for compatibility with v1 and v2. It may not be needed.